ecr/internal/testdata/digest.go

```diff
@@ -11,3 +25,13 @@ const (
 	// ImageDigest is used for consistent, placeholder image digests in tests.
 	ImageDigest digest.Digest = "image-digest"
 )
+
+// GenerateDigest returns a psuedo-random digest for use as a distinct digest in
+// tests. The produced value does not use a secure random number generator and
+// will give out deterministic values.
+func GenerateDigest() digest.Digest {
+	d := digest.SHA256.Digester()
+	fmt.Fprintf(d.Hash(), "%s", time.Now().UTC().Format(time.RFC3339Nano))
+	fmt.Fprintf(d.Hash(), "%d", generatorRand.Int())
+	return d.Digest()
+}
```

Comment:
This function is no longer used and can be removed.

Reply:
Nice catch! Removed in the latest push.